8.x/core-concepts/cruds.md

```diff
@@ -2,99 +2,55 @@
 
 [[toc]]
 
-This document reviews how to implement CRUD (create, read, update, delete) in Larammerce.
-As you know, it is not only related to Larammerce and is related to the structure of the Laravel framework.
-In larammerce, tools have been created to make implement in CRUD easier. Also, the created CRUD should be the same as the previous CRUDs of the system, so it should be followed the guidelines provided in this document.
+This document reviews how to implement `CRUD` (create, read, update, delete) in the Larammerce platform. A `ToDoList` project is done step by step to illustrate the operational concepts.
 
-In this tutorial, the principles of CRUD operation in Larammerce will be taught, so, let's follow a few steps to create it:
+#### Make migrations
 
-**Step 1**: Create Migration
+Larammerce is a schema-first platform. So you must create a table for your todos.
 
-**Step 2**: Add Model
+Run the following command in the terminal:
 
-**Step 3**: Add Controller
-
-**Step 4**: Add Veiws 
-
-::: warning WARNING
-The tutorials in this document are based on Laravel **version 8.75**.
-:::
-
-For example, a to-do list is created to show how to the operation.
-
-### Step 1: Create Migration
-Since the Laravel framework is a **schema-first** framework, so, the database schema must be created first.
-At this migration, so open a terminal or command line and run the following command:
-
-```bash
+```bash{1}
 php artisan make:migration create_todos_table
 ```
 
```

Comment:
After running this code, a file named  `<a date>_create_todos_table.php` will be created in the path `/larammerce/database/migrations` containing Some auto-generated functions like `schema:create` and `schema:dropIfExist`. To add some columns like "subject" and "status" to the database, insert the codes into the file as follows: